File: java/program/Clothing.java
public class Clothing extends Product {
    private String size;
    private String material;
    private String gender;

    public Clothing(String idProduct, String name, String brand, String price, String size, String material, String gender) {
        super(idProduct, name, brand, price);
        this.size = size;
        this.material = material;
        this.gender = gender;
    }

    // Getter and Setter Size
    public String getSize() {
        return size;
    }
    public void setSize(String size) {
        this.size = size;
    }

    // Getter and Setter Material
    public String getMaterial() {
        return material;
    }
    public void setMaterial(String material) {
        this.material = material;
    }

    // Getter and Setter Gender
    public String getGender() {
        return gender;
    }
    public void setGender(String gender){
        this.gender = gender;
    }
}

File: PHP/program/Clothing.php
<?php
/* Saya Nur Ainun 2202046 mengerjakan Latihan Praktikum 2 dalam mata kuliah Desain dan Pemrograman Berorientasi Objek 
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin. */

require_once('Product.php');
class Clothing extends Product
{
    private $size;
    private $material;
    private $gender;

    public function __construct($size = "", $material="", $gender="")
    {
        parent::__construct("", "", "", "");
        $this->size = $size;
        $this->material = $material;
        $this->gender = $gender;
    }

    // Getter and Setter for size
    public function get_Size()
    {
        return $this->size;
    }

    public function set_Size($size)
    {
        $this->size = $size;
    }

    // Getter and Setter for Material
    public function get_Material()
    {
        return $this->material;
    }

    public function set_Material($material)
    {
        $this->material = $material;
    }

    // Getter and Setter for Gender
    public function get_Gender()
    {
        return $this->gender;
    }

    public function set_Gender($gender)
    {
        $this->gender = $gender;
    }
}
?>